### PR TITLE
adding bert to onsite llms

### PR DESCRIPTION
**Pull request by ajitalapati**
Issue #69 : 

Gives ability to load in BERT models. however, fine-tuning needs to be added as well. I'm assuming it will have the same patten as the other models taken from HF but I'll do some actual testing later.

### CODE REVIEW

**Comment by ajn2004** [comment]
We need to add some code in src/llm_vm/client.py line 10
Notice how we have a diction of model name: model
we need to add
```python
"bert":llms.Small_Local_Bert,
```
in between lines 11 and 18.

From there we should be able to run quickstart.py by specifying 'bert' as the big_model in client and get generation with the bert model.

File: src/llm_vm/onsite_llm.py [hunk 1]
```diff
@@ -572,6 +574,50 @@ def generate(self,prompt,max_length=100,**kwargs): # both tokenizer and model ta
...
+        model_uri (str): Hugging Face Endpoint for LLM
+        tokenizer (AutoTokenizer): Tokenizer from Transformer's library
+        model (LLM): The large language model
+
+    Methods:
+        model_loader: Loads the LLM into memory
+        tokenizer_loader: Loads the tokenizer into memory
+        generate: Generates a response from a given prompt with the loaded LLM and tokenizer
+    """
+    def __init__(self,model_uri="bert-base-cased"): # tokenizer_kw_args=None,model_kw_args=None
```
[the remark this answers is not in the document]
Comment (ajitalapati) [reply src/llm_vm/onsite_llm.py:594]: It should be able to, do you have a specific model URI?

**Comment by cartazio** [comment]
https://huggingface.co/prajjwal1/bert-tiny

File: src/llm_vm/onsite_llm.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (cartazio) [reply src/llm_vm/onsite_llm.py:594]: https://huggingface.co/prajjwal1/bert-tiny

**Comment by cartazio** [comment]
@ajitalapati  if you look at the pythia model in main now, its become MUCH easier to add new models,

**Pull request closed without merging**